walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -22,59 +22,87 @@
 
 /** Finds potential meeting times. */
 public final class FindMeetingQuery {
+  public Collection<TimeRange> timesForEveryone;
+  public Collection<TimeRange> timesForConfirmed;
+  public int currentEndTime;
 
   /**
    * Returns available TimeRanges for the {@code request} given {@code events}, a Collection of
    * all events so far. 
    */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    Collection<TimeRange> times = new ArrayList<>();
-    Collection<String> attendees = request.getAttendees();
-    
+    timesForEveryone = new ArrayList<>();
+    timesForConfirmed = new ArrayList<>();
+
     if (request.getDuration() > TimeRange.WHOLE_DAY.duration()) {
       // If the event is longer than a day, no time slots are available to book a meeting.
-      return times; 
+      return new ArrayList<>();
```

Comment:
Sorry, I take back what I said. I think it makes more sense to return timesForEveryone as you are also setting this value.

Reply:
gotcha, updated.